src/views/CartView.vue

```diff
@@ -115,6 +118,16 @@ import ToastComponent from '@molgenis-ui/components/src/components/ToastComponen
 export default Vue.extend({
   name: 'CartView',
   components: { SpinnerAnimation, CollapseTreeIcon, ToastComponent },
+  beforeMount: function () {
+    const variableAssessments = {}
+
+    for (const [variableId, assessmentIds] of Object.entries(this.gridSelection)) {
```

Comment:
If the gridselection changes , this code is not run again, and variableAssessmenrs are not updated, this seams like an issue ?

Reply:
Ja, dat zou inderdaad kunnen gebeuren. Ik was in de veronderstelling dat gridSelection niet veranderd, maar dat gebeurd wel bij het laden van alle variabelen. Ik zet het nog een keer terug naar een computed property en kijk waarom het cachen niet wil. Dat is volgens mij de netste manier.

Reply:
or use method or use service and watch this.variableAssessments if these change then run the method ( or service call again)

Reply:
Currently there is no situation in the app where the variableAssessments change after mount ( that i can trigger ), i guess it's not really an issue